source/_docs/guides/jenkins.md

```diff
@@ -0,0 +1,267 @@
+
+    - Under **Run?** select **Regular expression match**.
+
+        - **Expression**: `(?i).*origin/master.*`
+
+        - **Label**: `${ENV,var="GIT_BRANCH"}`
+
+    - Under **Builder** select **Execute Shell**
+
+        - Command:
+
+                echo "Merging multi-dev changes to master"
+                terminus build:env:merge -n ${SITE_ID}.ci-${BUILD_ID} --yes
+
+    Your conditional step should look like this:
+    ![Conditional Step](/source/docs/assets/images/integrations/jenkins-conditional.png)
+
+6. Finally, a  cleanup task:
+
+    ```nohighlight
+    echo "Cleaning up multidev & branches"
+    git -C ${WORKSPACE} remote remove pantheon
+    git -C ${WORKSPACE} remote prune origin
+    terminus build:env:delete:ci ${SITE_ID} --keep=2 --delete-branch
+    ```
+
+### Add Post-build Actions
+
+Under **Post-build Actions** is another button labelled **Add post-build action**. Click on it and select the option "Set GitHub commit status (universal)". In the fields that appear, choose the following options:
+
+ - **Commit SHA**: "Latest build revision".
+
+ - **Repositories**: "Any defined in job repository".
+
+ - **Commit context**: "From GitHub property with fallback to job name".
+
+ - **Status Result**: "One of default messages and statuses".
+
+Finally, hit **Save** to complete the configuration of your Jenkins build process.
+
+## GitHub/Jenkins Integration
+
+1. From your GitHub account, go to [Settings](https://github.com/settings/profile). Under **Developer Settings** click on **Personal access tokens**  and generate a new token. This should have all **repo**  and **admin:repo-hook** options:
+
+    ![GitHub token permissions](/source/docs/assets/images/integrations/jenkins-gh-token-access.png)
+
+    Copy the generated token. Be careful, as you will not be able to view it again.
+
+2. Return to the main Jenkins dashboard and select **Manage Jenkins**, then **Configure System**.
+
+3. Scroll to the **GitHub Servers** section  and click on **Add GitHub Server**.
+
+4. Leave the **API URL** as "https://api.github.com." For **Credentials**, click **Add** then select the **Jenkins** user.
+
+5. In the pop-up menu, add the following settings:
+
+    - **Domain**: "Global credentials (unrestricted)
+    - **Kind**: "Secret Text"
+    - **Scope**: "Global (Jenkins, nodes, items, all child items, etc)
+    - **Secret**: Paste the GitHub token you created
+    - **ID** (Optional): An internal unique ID, if left blank, Jenkins will generate an ID
+    - **Description**: Optional description
+
+    When you're satisfied with these values, click **Add**. Now under the **Credentials** drown-down you can select your new credentials. Then **Save**.
+
+
+6. Now on your local computer, create a new branch and makes a change to it. When you commit and push to GitHub, a build should initiate. You should see the results of the test, which link to the Jenkins job:
```

Comment:
I think we're missing something, because I've completed this guide up to this point and verified that the GH token is working, but Jenkins will not initiate a build when I commit to a new branch.